front/lib/api/data_sources.ts

```diff
@@ -699,12 +700,14 @@ export async function createDataSourceWithoutProvider(
     space,
     name,
     description,
+    conversationId,
```

Comment:
Please use ConversationType vs a ModelId (which should ~never appear in interfaces)